### PR TITLE
render: temporary --disable-phase-2

### PR DESCRIPTION
**Pull request by sttts**
Counterpart of openshift/cluster-kube-apiserver-operator#91 and prerequisite for a https://github.com/openshift/cluster-kube-controller-manager-operator/pull/63 like change.

### CODE REVIEW

File: cmd/cluster-kube-scheduler-operator/render/render.go [hunk 1]
```diff
@@ -103,7 +113,21 @@ func (r *renderOpts) Run() error {
...
 
-	return genericrender.WriteFiles(&r.generic, &renderConfig.FileConfig, renderConfig)
+	// TODO: remove after the transition in the installer to a phase-2 free bootstrapping
+	var filters []assets.FileInfoPredicate
+	if r.disablePhase2 {
+		filters = append(filters, func(info os.FileInfo) bool {
+			if strings.HasPrefix(info.Name(), "kube-system-") {
+				return false
+			}
+			if info.Name() == "daemonset-kube-scheduler-daemonset.yaml" {
```
Comment (aveshagarwal) [inline cmd/cluster-kube-scheduler-operator/render/render.go]: i think name of the file here should match to `kube-scheduler-daemonset.yaml` as here https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/bindata/bootkube/manifests/kube-scheduler-daemonset.yaml ?

Comment (sttts) [reply cmd/cluster-kube-scheduler-operator/render/render.go]: indeed, typo

Comment (sttts) [reply cmd/cluster-kube-scheduler-operator/render/render.go]: fixed

**Comment by mfojtik** [comment]
/lgtm

This is just temporary until we transition the installer.

**Pull request merged**